### PR TITLE
docs: update readme file with the latest UI changes and comments

### PR DESCRIPTION
**Pull request by XiaofuHuang**

### CODE REVIEW

File: configs/Phi-3-mini-4k-instruct/README.md [hunk 1]
```diff
@@ -51,43 +51,38 @@ To use `prompt flow` in VS Code, please refer to this [Quick Start](https://micr
 
 ## Remote Development
 ### Prerequisites
-1. To run the model fine-tuning in your remote Azure Container App Environment, you need to make sure your subscription have enough GPU capacity amount. Submit a [support ticket](https://azure.microsoft.com/support/create-ticket/) to request the capacity amount required for your application. [Learn More about GPU capacity](https://learn.microsoft.com/en-us/azure/container-apps/workload-profiles-overview)
+1. To run the model fine-tuning in your remote Azure Container App Environment, your subscription has enough GPU capacity. Submit a [support ticket](https://azure.microsoft.com/support/create-ticket/) to request the required capacity for your application. [Get More Info about GPU capacity](https://learn.microsoft.com/en-us/azure/container-apps/workload-profiles-overview)
```
Comment (supkasar) [inline configs/Phi-3-mini-4k-instruct/README.md:54]: Add 'make sure' after comma, i.e. ...Environment, make sure your subscription has.....

File: configs/Phi-3-mini-4k-instruct/README.md [hunk 2]
```diff
@@ -51,43 +51,38 @@ To use `prompt flow` in VS Code, please refer to this [Quick Start](https://micr
...
 
 ### Run Fine-tuning
 To start the remote fine-tuning job, execute the `AI Toolkit: Run fine-tuning` command.
 
 To view the system and console logs, you can visit the Azure portal using the link in the output panel (more steps at [View and Query Logs on Azure](https://aka.ms/ai-toolkit/remote-provision#view-and-query-logs-on-azure)). Alternatively, you can view the console logs directly in the VSCode output panel by running the command `AI Toolkit: Show the running fine-tuning job streaming logs`. 
-> **Note:** The job might take a few minutes to initiate. If there is already a running job, the current one may be queued to start later.
-    
+> **Note:** The job might be queued due to insufficient resources. If the log is not displayed after execute the `AI Toolkit: Show the running fine-tuning job streaming logs` command, wait for a while and then execute the command again to re-connect to the streaming log.
```
Comment (supkasar) [inline configs/Phi-3-mini-4k-instruct/README.md:70]: Add 'comma' after displayed and remove 'after', i.e. .....If the log is not displayed, execute the 'AI Toolkit.....

File: configs/Phi-3-mini-4k-instruct/README.md [hunk 3]
```diff
@@ -51,43 +51,38 @@ To use `prompt flow` in VS Code, please refer to this [Quick Start](https://micr
...
 
 
 ### Deploy the Inference Endpoint
-If you wish to revise the inference code or reload the inference model, please execute the `AI Toolkit: Deploy for inference` command. This will synchronize your latest code with ACA and restart the replica.  
+If you wish to revise the inference code or reload the inference model, please execute the `AI Toolkit: Deploy for inference` command. This will synchronize your latest code with Azure Container App and restart the replica.  
 
-Once deployment is successfully completed, you can find the web API endpoint under `ACA_APP_ENDPOINT` within `./infra/inference.config.json`. You are now ready to evaluate the model using this endpoint.
+Once deployment is successfully completed, you can access the inference API by clicking on the "*Go to Inference Endpoint*" button displayed in the VSCode notification. Alternatively, the web API endpoint can be found under `ACA_APP_ENDPOINT` in `./infra/inference.config.json` and in the output panel. You are now ready to evaluate the model using this endpoint.
```
Comment (supkasar) [inline configs/Phi-3-mini-4k-instruct/README.md:85]: Replace 'alternatively' with 'or', i.e. Or, the web API endpoint....

File: configs/llama-v2-7b/README.md [hunk 4]
```diff
@@ -42,40 +42,40 @@ To use `prompt flow` in VS Code, please refer to this [Quick Start](https://micr
 
 ## Remote Development
 ### Prerequisites
-1. To run the model fine-tuning in your remote Azure Container App Environment, you need to make sure your subscription have enough GPU capacity amount. Submit a [support ticket](https://azure.microsoft.com/support/create-ticket/) to request the capacity amount required for your application. [Learn More about GPU capacity](https://learn.microsoft.com/en-us/azure/container-apps/workload-profiles-overview)
+1. To run the model fine-tuning in your remote Azure Container App Environment, your subscription has enough GPU capacity. Submit a [support ticket](https://azure.microsoft.com/support/create-ticket/) to request the required capacity for your application. [Get More Info about GPU capacity](https://learn.microsoft.com/en-us/azure/container-apps/workload-profiles-overview)
```
Comment (supkasar) [inline configs/llama-v2-7b/README.md:45]: Add 'make sure' after the comma, i.e. ....Environment, make sure your subscription......

File: configs/llama-v2-7b/README.md [hunk 5]
```diff
@@ -42,40 +42,40 @@ To use `prompt flow` in VS Code, please refer to this [Quick Start](https://micr
...
 
 ### Run Fine-tuning
 To start the remote fine-tuning job, execute the `AI Toolkit: Run fine-tuning` command.
 
 To view the system and console logs, you can visit the Azure portal using the link in the output panel (more steps at [View and Query Logs on Azure](https://aka.ms/ai-toolkit/remote-provision#view-and-query-logs-on-azure)). Alternatively, you can view the console logs directly in the VSCode output panel by running the command `AI Toolkit: Show the running fine-tuning job streaming logs`. 
-> **Note:** The job might take a few minutes to initiate. If there is already a running job, the current one may be queued to start later.
-    
+> **Note:** The job might be queued due to insufficient resources. If the log is not displayed after execute the `AI Toolkit: Show the running fine-tuning job streaming logs` command, wait for a while and then execute the command again to re-connect to the streaming log.
```
Comment (supkasar) [inline configs/llama-v2-7b/README.md:63]: Add comma after 'displayed' and remove 'after', i.e. If the log is not displayed, execute the...

File: configs/llama-v2-7b/README.md [hunk 6]
```diff
@@ -42,40 +42,40 @@ To use `prompt flow` in VS Code, please refer to this [Quick Start](https://micr
...
 
 
 ### Deploy the Inference Endpoint
-If you wish to revise the inference code or reload the inference model, please execute the `AI Toolkit: Deploy for inference` command. This will synchronize your latest code with ACA and restart the replica.  
+If you wish to revise the inference code or reload the inference model, please execute the `AI Toolkit: Deploy for inference` command. This will synchronize your latest code with Azure Container App and restart the replica.  
 
-Once deployment is successfully completed, you can find the web API endpoint under `ACA_APP_ENDPOINT` within `./infra/inference.config.json`. You are now ready to evaluate the model using this endpoint.
+Once deployment is successfully completed, you can access the inference API by clicking on the "*Go to Inference Endpoint*" button displayed in the VSCode notification. Alternatively, the web API endpoint can be found under `ACA_APP_ENDPOINT` in `./infra/inference.config.json` and in the output panel. You are now ready to evaluate the model using this endpoint.
```
Comment (supkasar) [inline configs/llama-v2-7b/README.md:78]: Replace 'Alternatively' with 'Or', ie. Or, the web API endpoint...

File: configs/Phi-3-mini-4k-instruct/README.md (hunk 1 above)
Comment (XiaofuHuang) [reply configs/Phi-3-mini-4k-instruct/README.md]: Updated.

File: configs/Phi-3-mini-4k-instruct/README.md (hunk 2 above)
Comment (XiaofuHuang) [reply configs/Phi-3-mini-4k-instruct/README.md]: Updated

File: configs/Phi-3-mini-4k-instruct/README.md (hunk 3 above)
Comment (XiaofuHuang) [reply configs/Phi-3-mini-4k-instruct/README.md]: Updated

File: configs/llama-v2-7b/README.md (hunk 4 above)
Comment (XiaofuHuang) [reply configs/llama-v2-7b/README.md]: Updated

File: configs/llama-v2-7b/README.md (hunk 5 above)
Comment (XiaofuHuang) [reply configs/llama-v2-7b/README.md]: Updated

File: configs/llama-v2-7b/README.md (hunk 6 above)
Comment (XiaofuHuang) [reply configs/llama-v2-7b/README.md]: Updated

**Review by supkasar — APPROVED** [review]

**Review by swatDong — APPROVED** [review]

**Pull request merged**